_config.yml

```diff
@@ -277,6 +277,9 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
+# youyan uid
```

Comment:
login uyan.cc
then get your code 
e.g.
```
<!-- UY BEGIN -->
<div id="uyan_frame"></div>
<script type="text/javascript" src="http://v2.uyan.cc/code/uyan.js?uid=34343324"></script>
<!-- UY END -->
```
then just fill your uid get from uyan.cc in the _config.yml
e.g.
youyan_uid=34343324